tycho-bundles/org.eclipse.tycho.p2.maven.repository/src/main/java/org/eclipse/tycho/repository/local/MirroringArtifactProvider.java

```diff
@@ -395,8 +361,7 @@ public boolean isFileAlreadyAvailable(IArtifactKey artifactKey) {
         }
         Map<String, String> map = new LinkedHashMap<>(descriptor.getProperties());
         //fix bad metadata in p2...
-        if (ArtifactTransferPolicy.isCanonicalFormat(descriptor)
-                && TychoConstants.PACK200_CLASSIFIER.equals(map.get(TychoConstants.PROP_CLASSIFIER))) {
```

Comment:
I think we should keep this here to only fix the metadata if pack200 classifier is found.